### DISCUSSION TITLE
tokio::runtime::Handle for 0.3

### DISCUSSION THREAD

**Discussion by tbraun96** [tokio-rs/tokio#2807] [General] [answered]
What is the equivalent for 0.3? I can't seem to find a cloneable ``Handle``

**Comment by nickkuk** [comment]
I think it will be `Arc<Runtime>`. But shutdown should be elaborated.

**Comment by Darksonn** [comment, marked as answer]
We are changing the `Runtime::block_on` method to take `&self` rather than `&mut self` in v0.3 in #2720, which removes the need for `Handle`. Check out the issue for more information.
